src/extension/ncwms/src/main/java/au/org/emii/geoserver/wms/GeoserverUrlIndex.java

```diff
@@ -30,12 +19,17 @@
 import org.opengis.filter.sort.SortBy;
 import org.opengis.filter.sort.SortOrder;
 
-public class UrlIndexFeatureSource implements UrlIndexInterface {
-    static Logger LOGGER = Logging.getLogger(UrlIndexFeatureSource.class);
+import java.io.IOException;
+import java.util.*;
```

Comment:
Ok I'm using Intellij. I remove this line and start importing one by one. After about 4 separate imports it wrote the `import java.util.*;` line again